packages/common/src/utils/environment-utils.ts

```diff
@@ -8,4 +8,11 @@ export class EnvironmentUtils {
   static useRustCeramic(): boolean {
     return process.env.IPFS_FLAVOR !== 'go'
   }
+
+  /**
+   * Changes the server from expecting rust or go ipfs to the other
+   */
+  static setIpfsFlavor(flavor: 'go' | 'rust') {
+    process.env.IPFS_FLAVOR = flavor
```

Comment:
I don't actually know enough about Node to know if this is a good or bad idea, but instead of overriding the env var could we have a file-local static variable we override instead and that takes priority in the `useRustCeramic` call above?

Reply:
it worked fine but setting an env var is maybe bad form so switched to a file variable